instance/tests/models/test_openedx_instance.py

```diff
@@ -191,6 +227,26 @@ def test_spawn_appserver(self, mocks, mock_provision):
         self.assertEqual(configuration_vars['EDXAPP_CONTACT_EMAIL'], instance.email)
 
     @patch_services
+    @patch('instance.models.openedx_instance.OpenEdXAppServer.provision', return_value=True)
+    def test_spawn_appserver_with_external_domains(self, mocks, mock_provision):
+        """
+        Test that external domains are used when provisioning a new server.
```

Comment:
@mtyaka "Test that relevant configuration variables use external domains when provisioning a new app server." perhaps?